docs/kubernetes_state_metrics.md

```diff
@@ -116,7 +115,6 @@ data:
         labels_mapper:
           label_service: service
           label_app: app
-        telemetry: true
 ``` 
 
 The above example will create 2 separate Cluster Level Checks, using different collectors and features (label joins, telemetry, remapping...).
```

Comment:
should we also remove it here?

Reply:
I think it shouldn't be documented in the first place